### PR TITLE
GEOPS-210: Support Prometheus metrics

### PR DESCRIPTION
**Pull request by JayH5**
TODO:
- [x] Test metrics are preserved across worker restarts or multiple workers
- [x] More documentation
- [x] ~Clean up config (maybe put it all in `/gunicorn_conf.py`?)~ (#102)

### CODE REVIEW

File: gunicorn_conf.py [hunk 1]
```diff
@@ -18,3 +18,34 @@
...
+    return DEFAULT_PROMETHEUS_MULTIPROC_DIR
+
+
+raw_env = ["=".join(("prometheus_multiproc_dir", _prometheus_multiproc_dir()))]
+
+
+def worker_exit(server, worker):
+    # Do the Prometheus multiprocess thing if its working directory is set and
+    # the library is importable.
+    # https://github.com/prometheus/client_python#multiprocess-mode-gunicorn
```
Comment (jerith) [inline gunicorn_conf.py]: I don't think this comment is accurate.

Comment (jerith) [reply gunicorn_conf.py]: I guess what I mean is that it doesn't really explain anything in the context of `worker_exit()`.

**Review by jerith — APPROVED** [review]
:+1: 🍰

**Comment by JayH5** [comment]
I want to test some images before merging this and I may rebase since there are a lot of commits here that are now in master.

**Comment by JayH5** [comment]
Superseded by #108

**Pull request closed without merging**